go.mod

```diff
@@ -62,3 +65,5 @@ require (
 	honnef.co/go/tools v0.0.1-2020.1.5 // indirect
 	upper.io/db.v3 v3.7.1+incompatible
 )
+
+replace github.com/pingcap/br => github.com/3pointer/br v0.0.0-20200817122845-968a9b5582e8
```

Comment:
remember to remove the go mod replace after https://github.com/pingcap/br/pull/463 is merged